include/boost/sync/detail/throw_exception.hpp

```diff
@@ -18,9 +18,6 @@
 #include <boost/throw_exception.hpp>
 #include <boost/preprocessor/seq/enum.hpp>
 #include <boost/sync/detail/config.hpp>
-#if defined(BOOST_EXCEPTION_DISABLE)
-#include <boost/current_function.hpp>
```

Comment:
`source_location.hpp` include is missing.

Reply:
Added.